api/db/changelog/db.changelog-39-add-cluster-retry-number.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="brubenst" id="changelog-39-add-cluster-retry-number">
+    <addColumn tableName="user">
+      <column name="cluster_retry_number" type="INTEGER" />
```

Comment:
naming nit: cluster_create_retries? The "_number" suffix doesn't feel quite right to me, possibly the redundancy with the column type, or slight ambiguity since "number" doesn't necessarily == count.

Including "create" would also be nice in case we need to retry anything else cluster related in the future.